app/src/main/java/de/tum/in/tumcampusapp/adapters/SimpleStickyListHeadersAdapter.java

```diff
@@ -0,0 +1,87 @@
+package de.tum.in.tumcampusapp.adapters;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.TextView;
+
+import java.util.List;
+
+import de.tum.in.tumcampusapp.R;
+import se.emilsjolander.stickylistheaders.StickyListHeadersAdapter;
+
+abstract public class SimpleStickyListHeadersAdapter<T extends SimpleStickyListHeadersAdapter.SimpleStickyListItem>
```

Comment:
So this class is then unifying common tasks that are pretty much done also in LecturesListAdapter? Can you use your SimpleStickyListHeadersAdapter to fix douplicate code in LecturesListAdapter then in separate PR please? Also add a comment to this class that explains what this common class is doing.